server/src/main/java/org/opensearch/rest/action/cat/RestIndicesAction.java

```diff
@@ -599,6 +599,12 @@ protected Table getTableWithHeader(final RestRequest request) {
         );
         table.addCell("pri.search.concurrent_query_total", "default:false;text-align:right;desc:total query phase ops");
 
+        table.addCell(
```

Comment:
@jed326 I think we've missed that before: if concurrent search experimental feature is disabled, we should not show these stats, right?

Reply:
Good question. I think that makes sense for the REST response to be gated behind the feature flag but the `SearchStats` object itself probably should not be ser/de without the new fields even when the flag is disabled since that seems like it will lead to problems.

So basically the APIs will just "hide" these concurrent search metrics when the feature flag is disabled but they will still be collected by each shard. What do you think?

Reply:
> So basically the APIs will just "hide" these concurrent search metrics when the feature flag is disabled but they will still be collected by each shard. What do you think?

Yes, I think it makes sense - we don't want user facing changes for disabled features

Reply:
Actually I see that some of the remote store changes are doing this where they do serialize/deserialize based on the feature flag:

https://github.com/opensearch-project/OpenSearch/blob/aca2e9de7daaadee77c33e21ff5215c3f1e8600f/server/src/main/java/org/opensearch/action/admin/cluster/snapshots/restore/RestoreSnapshotRequest.java#L155-L157

Reply:
:+1: , even better

Reply:
I will avoid FeatureFlag based check for ser-de of stats API instead will prefer to use it to control API response. Usually stats API will be queried by monitoring systems periodically and if a domain is moving from feature flag enabled to feature flag disabled and in same version then it can see ser/de issues.

Reply:
Makes sense to me, I'll skip the feature flag check for the streaminput/output logic then. Thanks!